lib/engine/game/g_18_al.rb

```diff
@@ -46,11 +54,86 @@ def operating_round(round_num)
         ], round_num: round_num)
       end
 
+      def stock_round
+        Round::Stock.new(self, [
+          Step::DiscardTrain,
+          Step::G18AL::BuySellParShares,
+        ])
+      end
+
+      def action_processed(action)
+        super
+        case action
+        when Action::Assign
```

Comment:
let's hold off on merging this pr until i finish the special action

Reply:
Have updated commit to use special action.